src/main/java/com/zufar/onlinestore/payment/service/impl/PaymentEventServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.zufar.onlinestore.payment.service.impl;
+
+import com.stripe.exception.SignatureVerificationException;
+import com.stripe.model.Event;
+import com.stripe.model.EventDataObjectDeserializer;
+import com.stripe.model.PaymentIntent;
+import com.zufar.onlinestore.payment.enums.PaymentStatus;
+import com.zufar.onlinestore.payment.processor.PaymentEventProcessor;
+import com.zufar.onlinestore.payment.service.PaymentEventService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentEventServiceImpl implements PaymentEventService {
+
+    private final PaymentEventProcessor paymentEventProcessor;
+
+    public void processPaymentEvents(String payload, String header) throws SignatureVerificationException {
```

Comment:
Give better more clear name for `header` please
header for what? header of what?

The same thing for `payload` please
payload of what?

Reply:
Done